<launch>
    <node pkg = "robotic_depowdering" type = "path_planner" name = "path_planner" output = "screen"/>
    
    <param name = "waypointSafetyDist1_cm" type = "double" value = "2.5"/>
    <param name = "waypointSafetyDist2_cm" type = "double" value = "2.0"/>
</launch>